sdk/cosmos/azure-cosmos/test/test_correlated_activity_id.py

```diff
@@ -0,0 +1,65 @@
+# The MIT License (MIT)
+# Copyright (c) 2022 Microsoft Corporation
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import unittest
+from unittest.mock import MagicMock
+
+import pytest
+
+import azure.cosmos.cosmos_client as cosmos_client
+import test_config
+from azure.cosmos.partition_key import PartitionKey
+
+pytestmark = pytest.mark.cosmosEmulator
+
+
+@pytest.mark.usefixtures("teardown")
+class CorrelatedActivityIdTest(unittest.TestCase):
```

Comment:
This looks like a unit test. Can we add a test for making sure correlatedActivityId comes back as part of response from backend. 
We should add a test that runs on either emulator or with live cosmos db account. 

That way we can also check for the diagnostics.

Reply:
Just a question on top of it, I see in Java, we actually exposed a method in FeedResponse to get the correlationId, do we need to expose in Python as well?

Reply:
We should, so that customers can actually use it. @JericHunter can you please follow up on this ask in this PR?

Reply:
Simon informed me that we don't have response objects the way Java does. We truncate the response as soon as we receive it and return only the object(s) requested. So basically, that ask is not available to be done @kushagraThapar.

Reply:
@JericHunter I looked at the code and I found out the usage of `response_hook` in our CRUD APIs. We use response_hook as a way to provide response headers to the end user. So we should be able to add a test where we basically check the correlatedActivityId in the response headers in case of query operations. 
Also, a test through diagnostics would be great as well to add.